scrapers_next/ut/committees.py

```diff
@@ -0,0 +1,76 @@
+from spatula import JsonListPage, URL
+import logging
+from openstates.models import ScrapeCommittee
+import requests
+
+
+class UnknownParentError(BaseException):
+    def __init__(self, com_name):
+        super().__init__(f"Parent unknown for subcommittee {com_name}")
+
+
+class CommitteeList(JsonListPage):
+    source = URL(
+        "https://le.utah.gov/data/committees.json",
+        timeout=10,
+    )
+
+    def process_page(self):
+        legislators_url = "https://le.utah.gov/data/legislators.json"
+        membership = {}
+        response = requests.get(legislators_url)
+        for each_legislator in response.json()["legislators"]:
```

Comment:
We call `response.json()` frequently. Any reason we can't call it _once_ before these loops and work on the output instead?

Reply:
Actually 1 time is from self & other is from requests I was trying different ways I thought I had lint issue Let me revert.